restcomm/restcomm.ussd/src/main/java/org/restcomm/connect/ussd/telephony/UssdCallManager.java

```diff
@@ -282,7 +282,7 @@ private ActorRef outbound(final Object message) throws ServletParseException {
         SipURI to = (SipURI)sipFactory.createSipURI(request.to(), uri);
 
         String transport = (to.getTransportParam() != null) ? to.getTransportParam() : "udp";
-        from = outboundInterface(transport);
+        //from = outboundInterface(transport);
```

Comment:
that is fine, I only removed that because the outboundInterface is actually never used. That also works for me.